portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/FDSViews.testcase

```diff
@@ -0,0 +1,146 @@
+			FrontendDataSetAdmin.createDataSet(
+				key_name = "Test Dataset",
+				key_type = "Channel");
+		}
+
+		task ("And Add a new view") {
+			FrontendDataSetAdmin.createDataSetView(
+				dataSetName = "Test Dataset",
+				description = "Description Test",
+				key_name = "View Test");
+		}
+
+		task ("And The user tries to delete the view") {
+			Click(
+				key_name = "View Test",
+				locator1 = "FrontendDataSetAdmin#DATA_SET_VIEW_ENTRY_KEBAB_MENU");
+
+			ClickNoError(locator1 = "ObjectPortlet#DELETE_ENTRY_BUTTON");
+
+			AssertElementPresent(
+				key_modalText = "Deleting a dataset view is an action that cannot be reversed. The content will be deleted and some dataset fragments may not be displayed.",
+				locator1 = "FrontendDataSetAdmin#DELETE_DATA_SET_MODAL_BODY");
+		}
+
+		task ("And When the user clicks on the 'X' button in the modal") {
+			Click(
+				key_modalTitle = "Delete Dataset View",
+				locator1 = "Button#CLOSE_MODAL");
+		}
+
+		task ("Then The dataset is present in the data set Admin page") {
+			AssertElementPresent(
+				entryName = "View Test",
+				locator1 = "FrontendDataSetAdmin#DATA_SET_VIEW_ENTRY_NAME");
+		}
+	}
+
+	@description = "LPS-180893. Confirm that a dataset view can be deleted"
+	@priority = 5
+	test CanDeleteView {
+		task ("When The user goes to add a new Dataset") {
+			FrontendDataSetAdmin.createDataSet(
+				key_name = "Test Dataset",
+				key_type = "Channel");
+		}
+
+		task ("And Add a new view") {
+			FrontendDataSetAdmin.createDataSetView(
+				dataSetName = "Test Dataset",
+				description = "Description Test",
+				key_name = "View Test");
+		}
+
+		task ("And the user deletes a View created") {
+			FrontendDataSetAdmin.deleteDataSetView(viewName = "View Test");
+		}
+
+		task ("Then Confirm that the Success message should appear") {
+			Alert.viewSuccessMessage();
+		}
+
+		task ("And The dataset View is not present in the data set Admin page") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Object",
+				panel = "Control Panel",
+				portlet = "Datasets");
+
+			AssertElementPresent(
+				locator1 = "FrontendDataSetAdmin#NUMBER_OF_VIEWS",
+				numberOfViews = 0);
+		}
+	}
+
+	@description = "LPS-180895  Confirm that the view was really deleted"
+	@priority = 5
+	test CanUpdateNumberOfViews {
+		task ("When The user goes to add a new Dataset") {
+			FrontendDataSetAdmin.createDataSet(
+				key_name = "Test Dataset",
+				key_type = "Channel");
+		}
+
+		task ("And Add a new view") {
+			FrontendDataSetAdmin.createDataSetView(
+				dataSetName = "Test Dataset",
+				description = "Description Test",
+				key_name = "View Test");
+		}
+
+		task ("And the user deletes a View created") {
+			FrontendDataSetAdmin.deleteDataSetView(viewName = "View Test");
+		}
+
+		task ("And back to the Datasets admin page") {
+			Navigator.gotoBack();
+		}
+
+		task ("Then confirm that the views column shows 0 views") {
+			AssertElementPresent(locator1 = "FrontendDataSetAdmin#VIEWS_NUMBER");
```

Comment:
```suggestion
			AssertElementPresent(
				locator1 = "FrontendDataSetAdmin#NUMBER_OF_VIEWS",
				numberOfViews = 0);
```

This step the way it is done does not confirm that the number is zero, only that the number is present.